src/app/pages/MarkdownPage.tsx

```diff
@@ -16,7 +17,16 @@ export type MarkdownPageProps = {
 export const MarkdownPage: React.FC<MarkdownPageProps> = (props): JSX.Element => {
     usePageTitle(props.title);
     useGoogleAnalyticsPageView();
-    useBackgroundColor(props.background);
+    const theme = useTheme();
+    if (props.background === 'light') {
+        useBackgroundColor(
+            theme.palette.type === 'light' ? theme.palette.background.paper : theme.palette.background.default
+        );
+    } else if (props.background === 'dark') {
+        useBackgroundColor(theme.palette.type === 'light' ? Colors.white[200] : theme.palette.background.default);
+    } else {
+        useBackgroundColor(props.background);
+    }
```

Comment:
I think this violates the rules of hooks (having a hook call in a conditional block). You may want to refactor this slightly to do the conditional logic to get the colors and then have a single call to useBackgroundColor after that.